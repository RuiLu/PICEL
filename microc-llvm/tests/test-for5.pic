int main() {
	int count = 0;
	for (int i = 0; i < 2; i=i+1) {
		prints("first loop:");
		print(i);
		count=count+1;
		for (int i = 2; i < 4; i=i+1) {
			prints("second loop:");
			print(i);
			count=count+1;
			for (int i = 4; i < 6; i=i+1) {
				prints("third loop:");
				print(i);
				count=count+1;
			}
		}
	}
	prints("count:");
	print(count);
}
