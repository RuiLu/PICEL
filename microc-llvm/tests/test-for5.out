first loop:
0
second loop:
2
third loop:
4
third loop:
5
second loop:
3
third loop:
4
third loop:
5
first loop:
1
second loop:
2
third loop:
4
third loop:
5
second loop:
3
third loop:
4
third loop:
5
count:
14
